### ISSUE TITLE
Invalid response for blob

### ISSUE THREAD

**Issue by jbcullis** [microsoft/react-native-windows#10036]
### Problem Description

Testing the migration from .66 to .68 and we are seeing a fetch error "Invalid response for blob".

**Further**
Issue also exists on .67 but a fresh install at .66 is ok.

Error: Invalid response for blob: eyJlcnJvciI6IlVuYWJsZSB0byByZXNvbHZlIG1vZHVsZSAuL2RlYnVnZ2VyLXVpL2RlYnVnZ2VyV29ya2VyLmFjYTE3M2M0IGZyb20gQzpcXFVzZXJzXFxqYW1lc1xcRGVza3RvcFxcU2l0ZU1lc2gvLjogLi9kZWJ1Z2dlci11aS9kZWJ1Z2dlcldvcmtlci5hY2ExNzNjNCBjb3VsZCBub3QgYmUgZm91bmQgd2l0aGluIHRoZSBwcm9qZWN0IG9yIGluIHRoZXNlIGRpcmVjdG9yaWVzOlxuICBub2RlX21vZHVsZXMifQ==
    at XMLHttpRequest.get (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:30545:21)
    at XMLHttpRequest.xhr.onload (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:25151:46)
    at XMLHttpRequest.dispatchEvent (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:31774:27)
    at XMLHttpRequest.setReadyState (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:30894:18)
    at XMLHttpRequest.__didCompleteResponse (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:30664:16)
    at index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:30811:47
    at EventEmitter.emit (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:2162:37)
    at MessageQueue.__callFunction (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:3128:31)
    at index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:2852:17
    at MessageQueue.__guard (index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:3079:13)

### Steps To Reproduce

Fresh install >
install react-native-async-storage

### Expected Results

Fetch should work

### CLI version

7.0.3

### Environment

```markdown
OS: Windows 10 10.0.22000
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Memory: 16.61 GB / 31.73 GB
  Binaries:
    Node: 16.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.17 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.1.3 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
      AllowAllTrustedApps: Enabled
      Versions: 10.0.18362.0, 10.0.19041.0, 10.0.22000.0
  IDEs:
    Android Studio: Not Found
    Visual Studio: 17.0.32126.317 (Visual Studio Professional 2022)
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.68.0 => 0.68.0
    react-native-windows: 0.68.4 => 0.68.4
  npmGlobalPackages:
    *react-native*: Not Found
```

### Target Platform Version

10.0.16299

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2022

### Build Configuration

Debug

**Comment by pegasusjcrawford** [comment]
Just wanted to add a +1 to this and add that I hit this error message on a freshly generated react-native-windows project using 0.68.x as my target.

**Comment by acoates-ms** [comment]
@JunielKatarn is this related to the blob work you just checked in?

**Comment by JunielKatarn** [comment]
> @JunielKatarn is this related to the blob work you just checked in?

(Just back from vacation).

Pretty sure it is.
In UWP apps, Blob should be currently disabled (need to discuss the property bag-passing mechanism first).

I'll start working on a fix today.

**Comment by JunielKatarn** [comment]
This is the commit I only applied to `0.66-stable`.
PRs for later branches including `main` will be published today.

https://github.com/microsoft/react-native-windows/pull/9983/commits/e25041698cd8b4ecf42c02c0457ea416fc8590db

**Comment by jbcullis** [comment]
Revisiting updating to latest: I'm now trying to go from .66 to .70 and this issue still exists.

**Comment by tero-paananen** [comment]
I have this error now on 0.69.11.  Issue did not exist on 0.69.7.

**Comment by JunielKatarn** [comment]
> I have this error now on 0.69.11. Issue did not exist on 0.69.7.

Looking into this.

**Comment by JunielKatarn** [comment]
@tero-paananen this is the most relevant related change between `0.69.7` and `0.69.11`.
It should prevent MSRN from attempting to instantiate the Blob module.

Can you please share the steps to reproduce?

<img width="924" alt="image" src="https://user-images.githubusercontent.com/4507319/199612378-2182f8da-66f6-48c5-833b-a51fc5e1509d.png">

**Comment by tero-paananen** [comment]
I reverted back into 0.69.7. It happened on application startup. I did 0.69.11 HelloWorld C# app by "react-native init" but that app did not has this error. I have to try 0.69.11 again on our own app soon...

**Comment by tero-paananen** [comment]
`Error: Invalid response for blob: xyz..` does not occur if i commented out BlobModule here

https://github.com/microsoft/react-native-windows/blob/00ff21a1f0bcd08f80706a33a04ce52c50e14f49/vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp#L65

but that is not valid fix

**Comment by tero-paananen** [comment]
> @tero-paananen Are you still hitting this? Should we open a fresh issue to capture all your up to date context?
> @JunielKatarn FYI

Yes, i have this issue still. I have to use 0.69.7 currently but i want to move on maybe 0.7x

**Comment by JunielKatarn** [comment]
Reopening.
Looking into it today.
